guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc

```diff
@@ -51,6 +51,12 @@ cred_store = keytab:/etc/httpd/conf/http.keytab
 cred_store = ccache:/var/lib/gssproxy/clients/krb5cc_%U
 euid = __ID_of_Apache_User__
 ----
+. Add a SELinux context rule to apply the appropriate security context to the `http.keytab` file:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# semanage fcontext -a -s system_u -t krb5_keytab_t '/etc/httpd/conf/http\.keytab'
```

Comment:
There was a suggestion to move the keytab to /etc/gssproxy, why keep it in /etc/httpd/conf when the purpose of this setup is to " implement stricter privilege separation for the Apache server by removing access to the keytab file while preserving Kerberos authentication functionality"?

Reply:
IMHO both locations (or any other) are good as long as correct permissions and SELinux context are set. I think with `/etc/gssproxy`, the semanage and restorecon commands may not be necessary.
Why not hold on to our own knowledge base though? https://access.redhat.com/articles/5854761

Reply:
Ok, let's stick with /etc/gssproxy to match the KBase article. I haven't dropped semanage because according to https://issues.redhat.com/browse/SAT-22855, it should be necessary. But I can drop it if you tell me to @lhellebr

I think I can resolve this thread for now.